src/test_l1_dsVideoDevice.c

```diff
@@ -513,13 +514,11 @@ void test_l1_dsVideoDevice_positive_dsGetDFC(void)
     for(int i = 0; i < gDSvideoDevice_NumVideoDevices; i++){
         result = dsGetVideoDevice(i, &handle);
         UT_ASSERT_EQUAL(result, dsERR_NONE);
-        UT_ASSERT_NOT_EQUAL(handle, (intptr_t)NULL);
 
         // Step 03: Get the DFC mode using dsGetDFC() with the obtained handle
+        result = dsGetDFC(handle, &dfc_mode);
         if(gSourceType == 1) {
-            result = dsGetDFC(handle, &dfc_mode);
             UT_ASSERT_EQUAL(result, dsERR_NONE);
-
             // Step 04: Compare the DFC mode with value from the profile
             for (int j = 0; j < gDSVideoDeviceConfiguration[j].NoOfSupportedDFCs; ++j) {
```

Comment:
update `i`
for (int j = 0; j < gDSVideoDeviceConfiguration[i].NoOfSupportedDFCs; ++j) {

check in L2 code also 
line:122

Reply:
Done